mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -334,6 +334,11 @@ function render_global_header( $attributes = array() ) {
 		$show_search  = true;
 	}
 
+	// Preload the menu font.
+	if ( is_callable( 'global_fonts_preload' ) ) {
```

Comment:
Will we need to add this `is_callable` every time we use it, or is this just for while making the PR?

Reply:
It was mostly for while PR'ing, but also for safety afterwards.

I guess it's safe to assume that everything that depends on the mu-plugin, will have the mu-plugins enabled, and a fatal when it doesn't is okay.

I'll strip out the is_callable's.